integration-tests/tests/multichain/actions/wait_for.rs

```diff
@@ -16,7 +16,7 @@ use near_primitives::views::FinalExecutionStatus;
 
 pub async fn running_mpc<'a>(
     ctx: &MultichainTestContext<'a>,
-    epoch: u64,
+    epoch: Option<u64>,
```

Comment:
why are we making this an Option when we don't use the `None` variant at all?

Reply:
We [do](https://github.com/near/mpc-recovery/pull/556/files#diff-88fb91ce851b88e0349a9b22dd5279e6cc6a590bb6ff1c2c604f3a4c8f70692bR82).
It is used when we do not know what epoch is now.